design/SAS/sas_gp_exp.R

```diff
@@ -0,0 +1,40 @@
+# Spatial exponential covariance structure ----
+formula <- FEV1 ~ cs(TIME | USUBJID)
```

Comment:
```suggestion
formula <- FEV1 ~ sp_exp(TIME | USUBJID)
```
please adjust example R code here